pricenode/README.md

```diff
@@ -67,6 +67,21 @@ curl http://localhost:8080/getVersion
 curl http://localhost:8080/info
 ```
 
+### Monitoring
+
+If you run a main pricenode, you also are obliged to activate the monitoring feed by running
```

Comment:
These "obligations" should be listed at https://bisq.wiki/Pricenode_Operator#Duties. I've just created that page as a stub. I've asked all @bisq-network/pricenode-operators to watch that page at https://github.com/bisq-network/roles/issues/14#issuecomment-591912853, so changes you make there will have a reasonable chance of being seen by them (duty changes should be communicated and followed up on more explicitly in any case, but FYI the wiki page is there, and we should rally around it).

Reply:
agreed. however, I do not have the authority to create directives. Creating directives can only be done by
- the project leader,
- the team leader of the team that is affected (the ops-team in this case), or
- the team leader leader
at the time of writing.

Reply:
Ok. By the powers vested in me as Admin team lead, I request that you transcribe what you've written here to the wiki, and as part of the overall delivery of task's parent project bisq-network/projects#13, announce that change to the `@bisq-network/pricenode-operators` team, and follow up that each operator actually complies. If you're not comfortable doing this, then please manage that @wiz does it in his role as ops team lead. The point is that changes like this are useless unless they're effectively communicated and followed up on. It doesn't mean you have to do the work yourself, but that you own that the work does get done. Updating a readme (or the wiki for that matter) on its own won't carry the day. Feel free to create an admin repo issue or issues about doing this documentation and communication work and request/suggest that @wiz be assigned.

Reply:
Note, the above should not hold up the merging of this PR.

Reply:
I've just updated the wiki to add running collectd as a "duty" with link to the README anchor that will exist after this PR is merged